### PR TITLE
Abort all connected up.  Closes #1253

### LINKED ISSUE #1253: Cromwell .20 returns 500 on abort
I ran a super small WDL, then checked its status to make sure it was running. Then, I tried to abort it -- the request returned a 500: "The server was not able to produce a timely response to your request.". However, when I check the status of the workflow, it says it has been successfully aborted.

I included the WDL i ran against https://cromwell.gotc-int.broadinstitute.org/swagger/index.html?url=/swagger/cromwell.yaml in case you'd like to try it yourself.

task echoHelloWorld {
	command {
		echo 'Hello, World!'
	}
	runtime {
		docker: "phusion/baseimage"
		disks: "local-disk 10 HDD"
		memory: "1 GB"
		preemptible: 3
	}
}

workflow printHelloAndGoodbye {
	call echoHelloWorld
}

### PR DESCRIPTION
**Pull request by mcovarr**
Addresses problems 1 and 2 from #1253, I'm still looking into problem 3.

### CODE REVIEW

File: engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala [hunk 1]
```diff
@@ -181,6 +186,11 @@ class WorkflowManagerActor(config: Config,
       data.idFromActor(workflowActor) foreach { workflowId =>
         jobStoreActor ! RegisterWorkflowCompleted(workflowId)
         workflowStore ! WorkflowStoreActor.RemoveWorkflow(workflowId)
```
Comment (Horneth) [inline engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: Won't the workflow already be removed from the WorkflowStore in the abort case here ?

Comment (mcovarr) [inline engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: yes, good catch!

File: engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala [hunk 2]
```diff
@@ -33,7 +32,7 @@ object WorkflowManagerActor {
     */
   sealed trait WorkflowManagerActorCommand extends WorkflowManagerActorMessage
   case object RetrieveNewWorkflows extends WorkflowManagerActorCommand
-  case class AbortWorkflowCommand(id: WorkflowId) extends WorkflowManagerActorCommand
+  case class AbortWorkflowCommand(id: WorkflowId, replyTo: ActorRef) extends WorkflowManagerActorCommand
```
Comment (geoffjentry) [inline engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: final

File: engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreActor.scala [hunk 3]
```diff
@@ -192,6 +201,8 @@ object WorkflowStoreActor {
   case class WorkflowsBatchSubmittedToStore(workflowIds: NonEmptyList[WorkflowId]) extends WorkflowStoreActorResponse
   case object NoNewWorkflowsToStart extends WorkflowStoreActorResponse
   case class NewWorkflowsToStart(workflows: NonEmptyList[WorkflowToStart]) extends WorkflowStoreActorResponse
+  case class WorkflowAborted(workflowId: WorkflowId) extends WorkflowStoreActorResponse
```
Comment (geoffjentry) [inline engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreActor.scala]: final on all case classes

**Comment by geoffjentry** [comment]
👍

**Comment by Horneth** [comment]
👍

**Pull request merged**